### DISCUSSION TITLE
Bookmark navigation macos

### DISCUSSION THREAD

**Discussion by bkarv** [alefragnani/vscode-bookmarks#401] [Q&A] [answered]
Thanks for the plugin been using it for a while. However I noticed in the recent updates when I toggle a bookmark it actually doesn't only toggle the bookmarks in the page but will jump to other pages with bookmarks. Is there a way to only toggle within the page? If so what is the macos command? Also is there away to clear all bookmarks? 

Thanks for the help.

**Comment by alefragnani** [comment, marked as answer]
Hi @bkarv ,

Are you saying that when you `toggle` a bookmark, it not only adds the bookmark, but also `jump to` another bookmark, and by _pages_ you mean _another document_?

I don’t remember seeing anything like it when developing the latest changes. In fact, the commands are quite different, and shouldn’t be affected by each other.

I suppose you are using keyboard shortcuts, so I suggest you to use the `Command Palette` to execute the commands, just to be sure the issue is with the command itself, not some shortcut conflict.

About _clearing all bookmarks_, you have two commands available, `Bookmarks: Clear` and `Bookmarks: Clear from All Files`.

Hope this helps
